blocks/OSC/src/osc/OscTypes.h

```diff
@@ -47,7 +47,7 @@ typedef std::int64_t	int64;
 typedef std::uint64_t	uint64;
 typedef std::int32_t	int32;
 typedef std::uint32_t	uint32;
-
+typedef std::uint8_t    uint8;

```

Comment:
It doesn't look like this new `typedef` is use and if so, it might be better to leave the file unchanged.